Source/ARTRealtimeChannel.m

```diff
@@ -605,12 +605,7 @@ - (void)transition:(ARTRealtimeChannelState)state status:(ARTStatus *)status {
             if (self.realtime.shouldSendEvents) {
                 channelRetryListener = [self unlessStateChangesBefore:self.realtime.options.channelRetryTimeout do:^{
                     [self.realtime.logger debug:__FILE__ line:__LINE__ message:@"RT:%p C:%p (%@) reattach initiated by retry timeout", self->_realtime, self, self.name];
-                    [self reattachWithReason:nil callback:^(ARTErrorInfo *errorInfo) {
-                        if (errorInfo) {
-                            ARTStatus *status = [ARTStatus state:ARTStateError info:errorInfo];
-                            [self setSuspended:status];
-                        }
-                    }];
+                    [self reattachWithReason:nil callback:nil];
```

Comment:
I think we can now remove the `callback` parameter from this method; it seems like nobody is using it any more.

Reply:
https://github.com/ably/ably-cocoa/pull/1551/commits/5ea06e1d219adae068b01fc2d467ffab4c6d983b